src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -362,6 +370,93 @@
         az netappfiles volume delete -g mygroup --account-name myaccname --pool-name mypoolname --name myvolname
 """
 
+helps['netappfiles volume replication'] = """
+type: group
+short-summary: Manage Azure NetApp Files (ANF) Volume replication operations.
+"""
+
+helps['netappfiles volume replication approve'] = """
+type: command
+short-summary: Authorize a volume as a replication destination for a specified source.
+parameters:
+  - name: --account-name -a
+    short-summary: The name of the ANF account
+  - name: --pool-name -p
+    short-summary: The name of the ANF pool
+  - name: --name --volume-name -n -v
+    short-summary: The name of the replication source volume
+  - name: --remote-volume-resource-id -d
+    short-summary: The resource id of the destination replication volume
```

Comment:
Can we both support destination volume name  and resource id here?

Reply:
The remote-volume-resource-id parameter expects the resource id.